### PR TITLE
Create setup.md

### PR DESCRIPTION
**Pull request by JasonTheKitten**
This is so that people know how to set it up

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+you will need to download the following Programs:
+IntelliJ
+Java 8 (You may need to uninstall Java 9)
+Java 8 SDK - You should choose the version that matches your Java
+git bash
+
+Run each of those, and wait for them to install.
+Next, open git bash and run the following:
+git clone you will need to download the following files:
```
Comment (lupus590) [inline setup.md]: i think you have a copy past error or something here

**Comment by SquidDev** [comment]
I'm going to be honest, I do feel this is overly complicated for what equates to:

 - Clone the repo
 - Run `./gradlew setupDecompWorkspace` in the directory
 - Import `build.gradle` into your editor of choice.

Have a look at some of Dan's comments on #290 as well - one doesn't need to spoon feed it to the user.

File: setup.md (hunk 1 above)
Comment (JasonTheKitten) [reply setup.md]: Yea I did have a copy/paste error. A few, in fact

**Comment by SquidDev** [comment]
@JasonTheKitten Have you ever used Gradle + a Java IDE before though? Once you've done it once it's pretty much the same for all projects: this repo doesn't need to be Java 101.

I think a better solution would be to have some variation of the above bullets and then link to http://mcforge.readthedocs.io/en/latest/gettingstarted/ for more information.

With respect to CC, we should probably mention `deploy.sh`, at least until we remove the need for it.

**Comment by JasonTheKitten** [comment]
I use windows cmd not linux bash so I can't use sh files

**Comment by JasonTheKitten** [comment]
> once you've used Gradle once, it's the same elsewhere.

What does that mean? If you mean it makes new gradle files, then can't we just ues copy/paste instead of the git command?

**Comment by SquidDev** [comment]
> > once you've used Gradle once, it's the same elsewhere.
> 
> What does that mean? 

Once you've set up Gradle in your editor once, it's pretty much the same procedure in any other project which uses Gradle.

**Comment by lupus590** [comment]
@JasonTheKitten are you aware that you still have this pull request open and you have made a commit against it which seems irrelevant to the pull request?

**Comment by JasonTheKitten** [comment]
That was an accident

**Comment by lupus590** [comment]
https://sethrobertson.github.io/GitFixUm/fixup.html

You will need to force the push, after following the steps in the above link, do this with `git push -f` (assuming you have your default remote set correctly)

**Comment by apemanzilla** [comment]
Force pushing is a bad idea. If you've already pushed erroneous commits (which you have) you should just revert the changes instead.

**Comment by KnightMiner** [comment]
Reverting leaves a revert commit which makes the PR much harder to review. Force push is perfectly valid for PRs

**Comment by apemanzilla** [comment]
The files changed tab will not show the difference because the changes are reverted, and the extra commits can be dropped when merging.

**Comment by thatcraniumguy** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**